### ISSUE TITLE
Double-clicking on inlay type hints on tuple unpacking shouldn't insert the hints

### ISSUE THREAD

**Issue by diceroll123** [microsoft/pylance-release#3095]
<h3> Environment data </h3>

-   Language Server version: 2022.7.41
-   OS and version: darwin x64
-   Python version (and distribution if applicable, e.g. Anaconda): Python 3.10 (for my example code)
-   python.analysis.indexing: null
-   python.analysis.typeCheckingMode: strict

<h3> Code Snippet </h3>

```python
def returns_tuple() -> tuple[int, str]:
    return 1, "Hello"

num, string = returns_tuple()
```

<h3> Repro Steps </h3>

1. Copy & Paste snippet into VSCode
2. Have inlay type hints enabled
3. Have **prerelease** PyLance installed
4. Double-click one of the inlay type hints for `num` or `string`

<h3> Expected behavior </h3>

I'd expect nothing to happen, I suppose. _However,_ having the inlay type hints there are still desirable.

<h3> Actual behavior </h3>

Type hints in those places break the code.

<h3> Logs </h3>

N/A

**Comment by bschnurr** [comment]
thanks, I'll disable the insert for tuples

**Comment by heejaechang** [comment]
This issue has been fixed in version 2022.7.42, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022742-27-July-2022)

**Comment by rishacha** [comment]
@heejaechang Now double-click for typing hints inlay doesn't work at all. 

![image](https://user-images.githubusercontent.com/30197357/187929281-3eec79af-db67-46b5-94e0-9a2199370f18.png)
